Tiltfile

```diff
@@ -186,6 +186,8 @@ def enable_provider(name):
     for substitution in substitutions:
         value = substitutions[substitution]
         yaml = yaml.replace("${" + substitution + "}", value)
+    if len(substitutions)==0:
+        fail("cant make k8s secret because there were NO kustomize_substitutions detected in tilt-settings.json")
```

Comment:
We should experiment shelling out to `clusterctl`, we might get better error reporting and (in the future) extra features like defaulting, etc